spec/fully_qualified_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Upsert do
+  it "works without a fully qualified name" do
+    upsert = Upsert.new $conn, :pets
+    assert_creates(Pet, [{:name => 'Jerry', :gender => 'male'}]) do
+      upsert.row({:name => 'Jerry'}, {:gender => 'male'})
+    end
+  end
+
+  it "works with a fully qualified name" do
+    table_name = ["#{RawConnectionFactory::DATABASE}2", :pets]
+    cls = clone_ar_class(Pet, table_name)
```

Comment:
perhaps define a new class Pet2 instead of metaprogramming?

Reply:
This seems to be working fine, maybe another thing we can address moving forward since we'll be removing `active_record_inline_schema` anyway?